### PR TITLE
February rke2 rcs 1.21 & 1.22

### PR DESCRIPTION
**Pull request by briandowns**

### CODE REVIEW

**Review by brandond — DISMISSED** [review]

File: channels-rke2.yaml
Comment (brandond) [inline channels-rke2.yaml]: This appears to be the same chart version as in `charts-v1-22-6-rke2r1`; I think you can just use that anchor without any changes or even creating a new anchor.

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by jossemargt-3pillar — DISMISSED** [review]

**Comment by rancher-max** [comment]
We also need to update the default version at the top before we merge this (with the non-rc version once we release that)

File: channels-rke2.yaml
Comment (brandond) [inline channels-rke2.yaml]: Actually it looks like it's changed: https://github.com/rancher/rke2/blob/release-1.22/Dockerfile#L102
```suggestion
        version: v3.21.4-build2022020801
```

File: channels-rke2.yaml
Comment (brandond) [inline channels-rke2.yaml]: https://github.com/rancher/rke2/blob/release-1.21/Dockerfile#L104
```suggestion
        version: v3.21.4-build2022020801
```

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by rancher-max — CHANGES_REQUESTED** [review]
It looks like we actually need to update `rke2-calico` version to `v3.21.401` on 1.22 as well:
https://github.com/rancher/rke2/blame/release-1.22/Dockerfile#L95-L112:

1.21 looks good:
https://github.com/rancher/rke2/blame/release-1.21/Dockerfile#L97-L116

**Review by brandond — APPROVED** [review]

**Review by rancher-max — APPROVED** [review]

**Pull request merged**
